reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md

```diff
@@ -0,0 +1,652 @@
+built to manage heterogeneous environments and hybrid cloud.
+
+In this release, we're introducing a number of new features, including:
+
+## Where can I install PowerShell?
+
+TODO: Refresh
+
+PowerShell 7 currently supports the following operating systems on x64, including:
+
+- Windows 8.1, and 10
+- Windows Server 2012, 2012 R2, 2016, and 2019
+- macOS 10.13+
+- Red Hat Enterprise Linux (RHEL) / CentOS 7
+- Fedora 30+
+- Debian 9
+- Ubuntu LTS 16.04+
+- Alpine Linux 3.8+
+
+Additionally, PowerShell 7.1 supports ARM32 and ARM64 flavors of Debian, Ubuntu, and ARM64 Alpine
+Linux.
+
+Check the installation instructions for your preferred operating system
+[Windows](/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7),
+[macOS](/powershell/scripting/install/installing-powershell-core-on-macos?view=powershell-7),
+or
+[Linux](/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-7).
+
+While not officially supported, the community has also provided packages for
+[Arch](https://aur.archlinux.org/packages/powershell/) and Kali Linux.
+
+> [!NOTE]
+> Debian 10 and CentOS 8 currently do not support WinRM remoting. For details on setting up
+> SSH-based remoting, see
+> [PowerShell Remoting over SSH](/powershell/scripting/learn/remoting/ssh-remoting-in-powershell-core?view=powershell-7).
+
+For more up-to-date information about supported operating systems and support lifecycle, see the
+[PowerShell Support Lifecycle](/powershell/scripting/powershell-support-lifecycle?view=powershell-7).
+
+## Running PowerShell 7.1
+
+TODO: Refresh + STORE
+
+PowerShell 7 installs to a new directory and runs side-by-side with Windows PowerShell 5.1. For
+PowerShell Core 6.x, PowerShell 7 is an in-place upgrade that removes PowerShell Core 6.x.
+
+- PowerShell 7 is installed to `%programfiles%\PowerShell\7`
+- The `%programfiles%\PowerShell\7` folder is added to `$env:PATH`
+
+The PowerShell 7 installer packages upgrade previous versions of PowerShell Core 6.x:
+
+- PowerShell Core 6.x on Windows: `%programfiles%\PowerShell\6` is replaced by
+  `%programfiles%\PowerShell\7`
+- Linux: `/opt/microsoft/powershell/6` is replaced by `/opt/microsoft/powershell/7`
+- macOS: `/usr/local/microsoft/powershell/6` is replaced by `/usr/local/microsoft/powershell/7`
+
+> [!NOTE]
+> In Windows PowerShell, the executable to launch PowerShell is named `powershell.exe`. In version 6
+> and above, the executable is changed to support side-by-side execution. The new executable to
+> launch PowerShell 7 is `pwsh.exe`. Preview builds will remain in-place as `pwsh-preview` instead
+> of `pwsh` under the 7-preview directory.
+
+## Improved shell experience with Predictive IntelliSense
+
+TODO
+
+## TODO - Add other Features
+
+## Breaking Changes and Improvements
+
+(TODO:IsComplete-ThruRC2)
+
+### Breaking Changes
+
+- **v7.1.0-preview.7**
```

Comment:
Remove these milestone labels. They are not relevant for the GA release. They all apply to GA. It does not matter what preview they came in.